### ISSUE TITLE
bug: CapacitorHttp throws exception on 204 HTTP response code

### ISSUE THREAD

**Issue by piotr-zurek** [ionic-team/capacitor#6250]
### Capacitor Version

```
Latest Dependencies:

  @capacitor/cli: 4.6.2
  @capacitor/core: 4.6.2
  @capacitor/android: 4.6.2
  @capacitor/ios: 4.6.2

Installed Dependencies:

  @capacitor/cli: 4.0.1
  @capacitor/core: 4.6.1
  @capacitor/android: 4.6.1
  @capacitor/ios: 4.6.2

```

### Platform(s)

iOS and Android

### Current Behavior

When 204 HTTP response is returned by API, Capacitor `fetch` proxy throw an error:
```
Uncaught (in promise) TypeError: Failed to construct 'Response': Response with null body status cannot have body
```

### Expected Behavior

Response is handled correctly.

### Code Reproduction

- setup project and enable CapacitorHttp
- build app and run Android/iOS emulator
- send HTTP request to any API which returns 204 HTTP code
- check browser console

### Other Technical Details

`npm --version` output:

`node --version` output:

`pod --version` output (iOS issues only):

### Additional Context

In my opinion this code in `native-bridge.js` is responsible for the error:
<img width="555" alt="image" src="https://user-images.githubusercontent.com/24444007/214555041-91bff16d-f34b-4041-9882-4c3e90d3672b.png">

`Response` constructor should have `null` as first argument for responses with 204 HTTP code.

**Comment by Ionitron** [comment]
This issue may need more information before it can be addressed. In particular, it will need a reliable Code Reproduction that demonstrates the issue.

Please see the Contributing Guide for [how to create a Code Reproduction](https://github.com/ionic-team/capacitor/blob/HEAD/CONTRIBUTING.md#creating-a-code-reproduction).

Thanks!
Ionitron 💙

**Comment by piotr-zurek** [comment]
@Ionitron here is the reproduction: https://github.com/piotr-zurek/capacitor-http-204-reproduction

1. Build app, sync and run on Android
2. Connect Chrome Dev Tools to instance
3. Click on "Send request" button
4. Check console

**Comment by piotr-zurek** [comment]
PR ready: https://github.com/ionic-team/capacitor/pull/6266

**Comment by piotr-zurek** [comment]
Any update on this? This issue is a blocker for me. Fix is ready in #6266 PR.

**Comment by jcesarmobile** [comment]
closing since the linked PR is merge
